_quarto.yml

```
@@ -118,6 +118,8 @@ website:
         contents:
           - text: "Workshop Setup"
             href: workshops/setup.md
+      - text: "Policies & Usage Costs" 
+        href: policies-usages/index.md              
```

There is no "s" in the directory name

```suggestion
        href: policies-usage/index.md              
```